dataframely/columns/datetime.py

```diff
@@ -329,16 +340,24 @@ def validation_rules(self, expr: pl.Expr) -> dict[str, pl.Expr]:
         return result
 
     def sqlalchemy_dtype(self, dialect: sa.Dialect) -> sa_TypeEngine:
+        timezone_enabled = self.time_zone is not None
         match dialect.name:
             case "mssql":
                 # sa.DateTime wrongly maps to DATETIME
-                return sa_mssql.DATETIME2(6)
+                return sa_mssql.DATETIME2(6, timezone=timezone_enabled)
             case _:
-                return sa.DateTime()
+                return sa.DateTime(timezone=timezone_enabled)
 
     @property
     def pyarrow_dtype(self) -> pa.DataType:
-        return pa.timestamp("us")
+        time_zone = (
+            self.time_zone.key
+            if isinstance(self.time_zone, ZoneInfo)
+            else self.time_zone.tzname(None)
+            if isinstance(self.time_zone, dt.timezone)
+            else self.time_zone
+        )
```

Comment:
This is out-of-date, I guess? `self.time_zone` cannot be `ZoneInfo` afaict

Reply:
Woops, I should have been more careful. I appreciate your careful, continued reviewing 🙏